src/transformers/GIPHY.js

```diff
@@ -0,0 +1,39 @@
+import { URL } from 'url';
+
+import { getTrimmedPathName, fetchOEmbedData } from './utils';
+
+const isMediaSubDomain = host => /^(media([0-9]{1,})?\.)giphy\.com$/.test(host);
+
+export const shouldTransform = url => {
+  const { host, pathname } = new URL(url);
+  const trimmedPathName = getTrimmedPathName(pathname);
+
+  return (
+    (['www.giphy.com', 'giphy.com'].includes(host) &&
+      trimmedPathName.includes('gifs/')) ||
```

Comment:
No need to use `trimmedPathName`, you can just use `pathname` instead
```suggestion
      pathname.includes('/gifs/')) ||
```